svir/dialogs/viewer_dock.py

```diff
@@ -785,6 +849,31 @@ def draw_losses_by_asset_aggr(self):
         table.resizeColumnsToContents()
         self.table_layout.addWidget(table)
 
+    def draw_avg_losses_stats_aggr(self):
+        self.plot_canvas.hide()
+        clear_widgets_from_layout(self.table_layout)
+        losses_array = self.avg_losses_stats_aggr['array']
+        losses_array = self._to_2d(losses_array)
+        tags = None
+        try:
+            tags = self.avg_losses_stats_aggr['tags']
+        except KeyError:
+            pass
```

Comment:
A comment says when the `pass` case happens would be useful. Why the 'tags' are missing sometimes?

Reply:
I found out that tags are needed (and available) only when the '*' is selected. In that case, we display them in the headers of the table, as row names. I've added some comments to clarify this.